ql/src/java/org/apache/hadoop/hive/ql/exec/tez/TezTask.java

```diff
@@ -671,6 +684,7 @@ DAGClient submit(DAG dag, Ref<TezSessionState> sessionStateRef) throws Exception
     }
 
     perfLogger.perfLogEnd(CLASS_NAME, PerfLogger.TEZ_SUBMIT_DAG);
+    runtimeContext.initFromTezClient(sessionState.getSession());
```

Comment:
minor: why not simply `runtimeContext.init(sessionState.getSession())`

Reply:
agreed, let me rewrite (I think in the early implementation phase I thought there will be different init methods), init looks cleaner